### PR TITLE
Add in a speed factor

### PR DESCRIPTION
**Pull request by jthielen**
In working on the ISU AMS Chase Sim this year, I found we were having issues timing issues in regard to the sails/meso-sails scans. They were not sped up by our simulation's speed factor, which caused the product times to leap too far forward and then backward in time, thus making our warnings and time-based placefiles go a little haywire.

To fix this, I tried to add in a speed factor argument to the l2munger code. It appears to be working in my simple test run, and we're planning on using this in our chase sim. Let me know if this is something you'd want to add into the master branch, or if you have any recommendations for how it could be improved.

Thanks!

### CODE REVIEW

**Comment by akrherz** [comment]
Thanks, but where's the actual code logic that uses this value?

File: l2munger.cpp [hunk 1]
```diff
@@ -247,17 +247,21 @@ time_t targetVolumeTime;
...
-  time_t seconds;  
+  time_t seconds;
+  time_t innerTimeDelta;  
 
   seconds = W88TimeToTime( w88_days, w88_seconds );
 
-  seconds += timeDelta;
+  innerTimeDelta = seconds - volumeTime;
+
+  seconds += timeDelta - innerTimeDelta + (innerTimeDelta / speedFactor);
```
Comment (jthielen) [inline l2munger.cpp:264]: Here is where the value is used...it just makes the time between the volume time and the product time be `speedFactor` times faster.

Comment (akrherz) [reply l2munger.cpp:264]: strange, glossed over that.

**Comment by jthielen** [comment]
One thing I forgot in this PR is to update the README. Would you want me to do that, or do you want to?

**Comment by akrherz** [comment]
@jthielen I'll do that, thank you

**Pull request merged**